stable/drone/values.yaml

```diff
@@ -80,15 +80,17 @@ sourceControl:
   secret:
   ## Fill in the correct values for your chosen source control provider
   ## Any key in this list with the suffix `Key` will be fetched from the
-  ## secret named above, if not provided the secret will default to
-  ## `<fullName>-source-control`
+  ## secret named above (you need to create in advance), if not provided the secret will default to
+  ## `<fullName>-source-control` and use the "ClientSecretKey:""secret" fields to create it. (if then, be awere to not share shis file with your password on it)
   github:
     clientID:
     clientSecretKey: clientSecret
+    Secret:
     server: https://github.com
   gitlab:
     clientID:
     clientSecretKey: clientSecret
+    Secret:
```

Comment:
Normally camel case for variables is used with a lowercase first letter. I would also use a better naming convention like `secretValue` or `clientSecretValue`.